### ISSUE TITLE
Can i use a standard ethernet network as an uplink with dynamic ip?

### ISSUE THREAD

**Issue by blackbadgerltd** [drsound/fault_tolerant_router#19]
Hi, i have 2 ethernet uplink interfaces i wish to balance (both from office LANs). The address assignment is dhcp. Will specifying 'pop' as the type get an address from the dhcp server, or is this only for pop connections?

Cheers

**Comment by drsound** [comment]
No, **ppp** triggers some specific checks for PPP interfaces, it wouldn't work with regular ethernet interfaces. Moreover Fault Tolerant Router requires the default route not to be specified in regular routing tables: if you use DHCP for sure you will get a default route too!
I will look into this issue, but it wouldn't be easy to tell a DHCP daemon not to set the new default route when the IP address is refreshed... ;-)
Meanwhile I suggest you to use some external script (possibly triggered by the DHCP daemon on the refresh event) to delete the regular default route, set the new IP addresses in _/etc/fault_tolerant_router.conf_, restart Fault Tolerant Router.
